Docs/source/install/hpc/dane.rst

```diff
@@ -58,13 +60,18 @@ Exit the ``vi`` editor with ``Esc`` and then type ``:wq`` (write & quit).
 
 .. important::
 
-   Now, and as the first step on future logins to Dane, activate these environment settings:
+   Now, and as the first step on future logins to Dane, activate these environment settings by executing the file:
 
    .. code-block:: bash
 
       source $HOME/dane_warpx.profile
 
-Finally, since Dane does not yet provide software modules for some of our dependencies, install them once:
+Finally, since Dane does not yet provide software modules for some of our dependencies, WarpX provides a script to install them.
+This is done executed now.
+They are by default installed in the workspace directory (which is recommended), but can be installed elsewhere by setting the environment variable ``WARPX_SW_DIR``.
```

Comment:
Probably sensible to generally rename from `SW_DIR` to `WARPX_SW_DIR` in all scripts...

Reply:
Fix in #6010